pkg/internal/cluster/create/create.go

```diff
@@ -190,12 +191,20 @@ func printUsage(name string) {
 			name,
 		)
 	} else {
+		// Detect shell and usage build prompt appropriately.
+		setKubeConfigPrompt := ""
+		switch shell := env.GetLinuxShell(); shell {
+		case "fish":
+			setKubeConfigPrompt = "set -x KUBECONFIG (kind get kubeconfig-path --name=%q)"
```

Comment:
I'm a bit dumbfounded that a Linux shell doesn't support POSIX... 🧐
https://pubs.opengroup.org/onlinepubs/009695399/utilities/export.html

Reply:
Yeah it's a bit strange but it appears to be the case. https://fishshell.com/docs/current/tutorial.html#tut_exports